simulate/simulate_xr/README.md

```diff
@@ -0,0 +1,71 @@
+# Simulate XR
+
+Simulate XR is an extension of `simulate` that renders the scene for VR/XR/AR headsets.
+
+
+## Installation
+
+To build simulate with XR, switch the option in [simulate CMakeLists](/simulate/CMakeLists.txt#L56) to ON, then configure, build and install normally. I prefer to keep it in a separate location from usual build and install, because launching with built-in XR takes a bit longer. The XR capability is also fully integrated into `libsimulate`, so should be usable from the loaded library or python bindings (untested).
```

Comment:
Its probably better to refer to the option by name here, this line will probably get out of sync since there's no proper way to know when it changes, OTOH renaming the option will find the usage in this readme